controller/test-files/deployments_osio/namespace-by-type.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
The name of this file is not following the convention we have elsewhere. Please name golden files like this:

```
<TEST-CASE>.<req|res>.<headers|payload>.golden.json
```

So in you case this would be `namespace-by-type.res.payload.golden.json` I guess. Please also correct the other files and try to include some headers as golden files. This ensures we see changes made to the headers of an endpoint by changes in the golden files.